sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1162,8 +1161,9 @@ private void executeQuery(
 
     private JobConfigurationQuery createBasicQueryConfig() {
       return new JobConfigurationQuery()
-          .setQuery(query.get())
           .setFlattenResults(flattenResults)
+          .setPriority("BATCH")
```

Comment:
Can you say more? This line is moved from L1152 in "startQueryJob".

We should use the same priority everywhere. priority is ignored for dry run.